### PR TITLE
Add language identifiers to code blocks

### PR DESCRIPTION
**Pull request by mikkelbu**
Relates to #2192

### CODE REVIEW

**Review by mairaw — CHANGES_REQUESTED** [review]
Thank you @mikkelbu. looks great. Left you a few comments for you to consider.

**Comment by kvnndr0** [comment]
On 30 Nov 2018 2:22 am, Mikkel Nylander Bundgaard <notifications@github.com> wrote:

Relates to #2192<https://github.com/dotnet/docs/issues/2192>

________________________________
You can view, comment on, or merge this pull request online at:

  https://github.com/dotnet/docs/pull/9313

Commit Summary

  *   Add language identifiers to code blocks

File Changes

  *   M docs/csharp/programming-guide/concepts/assemblies-gac/how-to-determine-if-a-file-is-an-assembly.md<https://github.com/dotnet/docs/pull/9313/files#diff-0> (2)
  *   M docs/framework/data/adonet/dataadapter-parameters.md<https://github.com/dotnet/docs/pull/9313/files#diff-1> (4)
  *   M docs/framework/data/adonet/downloading-sample-databases-linq-to-dataset.md<https://github.com/dotnet/docs/pull/9313/files#diff-2> (2)
  *   M docs/framework/data/adonet/generating-commands-with-commandbuilders.md<https://github.com/dotnet/docs/pull/9313/files#diff-3> (2)
  *   M docs/framework/data/adonet/modifying-data-with-stored-procedures.md<https://github.com/dotnet/docs/pull/9313/files#diff-4> (2)
  *   M docs/framework/data/adonet/ref-cursor-examples.md<https://github.com/dotnet/docs/pull/9313/files#diff-5> (4)
  *   M docs/framework/data/adonet/retrieving-data-using-a-datareader.md<https://github.com/dotnet/docs/pull/9313/files#diff-6> (2)
  *   M docs/framework/data/adonet/side-by-side-execution.md<https://github.com/dotnet/docs/pull/9313/files#diff-7> (4)
  *   M docs/framework/data/adonet/sql-server-connection-pooling.md<https://github.com/dotnet/docs/pull/9313/files#diff-8> (2)
  *   M docs/framework/data/adonet/sqlclient-streaming-support.md<https://github.com/dotnet/docs/pull/9313/files#diff-9> (10)
  *   M docs/framework/data/adonet/updating-data-sources-with-dataadapters.md<https://github.com/dotnet/docs/pull/9313/files#diff-10> (4)
  *   M docs/framework/data/transactions/committing-a-transaction-in-single-phase-and-multi-phase.md<https://github.com/dotnet/docs/pull/9313/files#diff-11> (6)
  *   M docs/framework/data/wcf/how-to-manually-generate-client-data-service-classes-wcf-data-services.md<https://github.com/dotnet/docs/pull/9313/files#diff-12> (4)
  *   M docs/framework/debug-trace-profile/bindingfailure-mda.md<https://github.com/dotnet/docs/pull/9313/files#diff-13> (2)
  *   M docs/framework/debug-trace-profile/callbackoncollecteddelegate-mda.md<https://github.com/dotnet/docs/pull/9313/files#diff-14> (8)
  *   M docs/framework/debug-trace-profile/code-contracts.md<https://github.com/dotnet/docs/pull/9313/files#diff-15> (4)
  *   M docs/framework/debug-trace-profile/dangerousthreadingapi-mda.md<https://github.com/dotnet/docs/pull/9313/files#diff-16> (2)
  *   M docs/framework/debug-trace-profile/datetimeinvalidlocalformat-mda.md<https://github.com/dotnet/docs/pull/9313/files#diff-17> (10)
  *   M docs/framework/debug-trace-profile/how-to-use-tracesource-and-filters-with-trace-listeners.md<https://github.com/dotnet/docs/pull/9313/files#diff-18> (4)
  *   M docs/framework/debug-trace-profile/illegalprepareconstrainedregion-mda.md<https://github.com/dotnet/docs/pull/9313/files#diff-19> (2)
  *   M docs/framework/debug-trace-profile/invalidapartmentstatechange-mda.md<https://github.com/dotnet/docs/pull/9313/files#diff-20> (2)
  *   M docs/framework/debug-trace-profile/jitcompilationstart-mda.md<https://github.com/dotnet/docs/pull/9313/files#diff-21> (2)
  *   M docs/framework/debug-trace-profile/loadfromcontext-mda.md<https://github.com/dotnet/docs/pull/9313/files#diff-22> (2)
  *   M docs/framework/debug-trace-profile/memberinfocachecreation-mda.md<https://github.com/dotnet/docs/pull/9313/files#diff-23> (2)
  *   M docs/framework/debug-trace-profile/opengenericcercall-mda.md<https://github.com/dotnet/docs/pull/9313/files#diff-24> (2)
  *   M docs/framework/debug-trace-profile/performance-counters.md<https://github.com/dotnet/docs/pull/9313/files#diff-25> (2)
  *   M docs/framework/debug-trace-profile/reentrancy-mda.md<https://github.com/dotnet/docs/pull/9313/files#diff-26> (2)
  *   M docs/framework/debug-trace-profile/releasehandlefailed-mda.md<https://github.com/dotnet/docs/pull/9313/files#diff-27> (2)
  *   M docs/framework/debug-trace-profile/virtualcercall-mda.md<https://github.com/dotnet/docs/pull/9313/files#diff-28> (2)
  *   M docs/framework/deployment/in-process-side-by-side-execution.md<https://github.com/dotnet/docs/pull/9313/files#diff-29> (4)
  *   M docs/framework/docker/console.md<https://github.com/dotnet/docs/pull/9313/files#diff-30> (4)

Patch Links:

  *   https://github.com/dotnet/docs/pull/9313.patch
  *   https://github.com/dotnet/docs/pull/9313.diff

**Comment by mairaw** [comment]
Did you mean to say something @kvnndr0?

File: docs/framework/data/adonet/generating-commands-with-commandbuilders.md [hunk 1]
```diff
@@ -76,7 +76,7 @@ builder.QuoteSuffix = "]";
   
  The following code example writes to the console the update command that was automatically generated.  
   
-```  
+```csharp
```
[the remark this answers is not in the document]
Comment (mikkelbu) [reply docs/framework/data/adonet/generating-commands-with-commandbuilders.md]: I've changed it to vb and added a second code block

File: docs/framework/debug-trace-profile/code-contracts.md [hunk 2]
```diff
@@ -51,7 +51,7 @@ Code contracts provide a way to specify preconditions, postconditions, and objec
   
  When `if`-`then`-`throw` statements appear in this form, the tools recognize them as legacy `requires` statements. If no other contracts follow the `if`-`then`-`throw` sequence, end the code with the <xref:System.Diagnostics.Contracts.Contract.EndContractBlock%2A?displayProperty=nameWithType> method.  
   
-```  
+```
```
[the remark this answers is not in the document]
Comment (mikkelbu) [reply docs/framework/debug-trace-profile/code-contracts.md]: Added csharp and also changed a lot of inline code samples into code blocks (they were already in separate paragraphs) to allow for syntax highlighting. Also, I've corrected some formatting and one example (with inspiration from https://www.macs.hw.ac.uk/~hwloidl/Courses/F21SC/CodeContracts_UserManual.pdf)

**Review by mairaw — APPROVED** [review]
Changes look great @mikkelbu. Thank you!

**Pull request merged**